kapitan/dependency_manager/helm/build.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+cd $(dirname "$0")
+pwd
+
+so_name="helm_fetch.so"
+
+# Compile the binding if a Go runtime exists
+if [[ -z $(which go) ]]; then
+    echo "[WARN] go is not available on this system -- skipping Helm binding build!"
+else
+    go build -buildmode=c-shared -o $so_name helm_fetch.go
+fi
+
+# Validate that the compiled binding exists
+if [[ -e $so_name ]]; then
+    echo "[INFO] $so_name built successfully or already exists"
+else
+    echo "[ERROR] $so_name does not exist!"
+    exit 1
+fi
+
+# Compile the Python ffi binding if Python is available
+if [[ -z $(which python3) ]]; then
+    echo "[WARN] python3 is not available on this system -- skipping cffi build!"
```

Comment:
Since these build steps are part of the CI/CD kapitan docker image, I'd say if this fails to build, the script should fail completely (?). That way travis correctly flags the build as broken. Otherwise the .so file won't build and the final released artifact will partially fail.

Reply:
Agreed. I was trying to stay consistent with the build script `kapitan/inputs/helm/build.sh` where I drew inspiration for using cgo in the PR. I'll add a `set -euo pipefail` at the top of this script.